source/codegen/common_helpers.py

```diff
@@ -34,7 +34,7 @@ def is_unsupported_struct(parameter):
 def is_unsupported_scalar_array(parameter):
   if not is_array(parameter['type']):
     return False
-  return is_enum(parameter) or get_underlying_type_name(parameter['type']) in {'ViInt16', 'ViBoolean'}
+  return is_enum(parameter) or get_underlying_type_name(parameter['type']) in {'ViInt16'}
```

Comment:
```suggestion
  return is_enum(parameter) or get_underlying_type_name(parameter['type']) == 'ViInt16'
```

Reply:
Ah, that's such a dummy mistake. I will raise a PR for this. Thanks. :)